requirements.txt

```diff
@@ -1,6 +1,6 @@
 scipy>=0.13.3
 numpy>=1.13.3
-pandas>=0.24.1,<1.1.0
+pandas>=1.0.0
```

Comment:
are there any changes that require using 1.0 or higher?

Reply:
Yes - the changes in `wrangle.py` don't work with older versions because of the way the date offsets are handled. There might be a way to come up with something that would work with both, but I'd have to spend some more time looking into it.

Reply:
Might be able to use a different `if-else` construct to check for the two scenarios that were present in the previous code. I'll check.